### PR TITLE
Feature/cookies and policies

### PR DESCRIPTION
**Pull request by johncordeiro**
This PR enable privacy policies, terms of use and cookie policy.

### CODE REVIEW

File: static/less/ureport2.less [hunk 1]
```diff
@@ -89,6 +89,56 @@ body {
   }
 }
 
+#ureport-policies-bar{
```
Comment (johncordeiro) [inline static/less/ureport2.less]: Some cases have a space between the id and the `{` and others don't. What's the standard?

File: static/less/ureport2.less [hunk 2]
```diff
@@ -1411,6 +1461,12 @@ ul.poll-filters, .featured-stories-filters, ul.stories-filters {
   font-size: .8em;
 }
 
+.policy-details {
+
```
Comment (johncordeiro) [inline static/less/ureport2.less]: empty line

File: templates/frame.haml [hunk 3]
```diff
@@ -367,6 +370,26 @@
...
   - block navbar
     - if org
+      -if org|check_policy:"cookie" or org|check_policy:"privacy" or org|check_policy:"tos"
+        %div#ureport-policies-bar{ style: 'border-color: {{ org|config:"primary_color" }}' }
+          %div.content
+            - trans "This website uses cookies to give you an incredible experience. By using this website you agree to the"
+            -if org|check_policy:"cookie"
+              %a{ href:"{% url 'public.policies' 'cookie' %}" }
+                - trans "Cookie Policy"
+              ,
```
Comment (johncordeiro) [inline templates/frame.haml]: In the case of having just Cookie Policy, we would have "Cookie Policy," at the end of the text.

**Review by johncordeiro — COMMENTED** [review]
fixing this, we're ready to go

File: static/less/ureport2.less (hunk 1 above)
Comment (edudouglas) [reply static/less/ureport2.less]: Done. U-Report doesn't have a standard.

File: static/less/ureport2.less (hunk 2 above)
Comment (edudouglas) [reply static/less/ureport2.less]: Done.

File: templates/frame.haml (hunk 3 above)
Comment (edudouglas) [reply templates/frame.haml]: Updated, please test again.

**Pull request closed without merging**